### PR TITLE
Feature/ban

### PR DESCRIPTION
**Pull request by otowa-kotori**

### CODE REVIEW

**Comment by otowa-kotori** [comment]
ItemListActivity started SetBlockListActivity will not respond to home ActionBar click

File: NewsApp/.idea/inspectionProfiles/Project_Default.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<component name="InspectionProjectProfileManager">
```
Comment (twd2) [inline NewsApp/.idea/inspectionProfiles/Project_Default.xml:1]: ???

File: NewsApp/.idea/inspectionProfiles/profiles_settings.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<component name="InspectionProjectProfileManager">
```
Comment (twd2) [inline NewsApp/.idea/inspectionProfiles/profiles_settings.xml:1]: ???

File: NewsApp/app/src/main/java/com/java/a35/newsapp/App.java [hunk 3]
```diff
@@ -28,6 +31,13 @@ public void onCreate() {
         api = new API(API.SERVER_URL, cachedLoader);
         pictureApi = new PictureAPI(cachedLoader);
         db = new StorageDbHelper(this);
+        Thread.UncaughtExceptionHandler oldHandler = Thread.getDefaultUncaughtExceptionHandler();
+        Thread.setDefaultUncaughtExceptionHandler(new Thread.UncaughtExceptionHandler() {
+            @Override
+            public void uncaughtException(Thread t, Throwable e) {
+                Log.e("test", Log.getStackTraceString(e));
+            }
+        });
```
Comment (twd2) [inline NewsApp/app/src/main/java/com/java/a35/newsapp/App.java:40]: ??????

**Review by twd2 — APPROVED** [review]

File: NewsApp/app/src/main/java/com/java/a35/newsapp/App.java (hunk 3 above)
Comment (otowa-kotori) [reply NewsApp/app/src/main/java/com/java/a35/newsapp/App.java:40]: 😁😭 For debugging purpose

**Pull request merged**